packages/kbn-grid-layout/grid/grid_row.tsx

```diff
@@ -201,25 +232,13 @@ export const GridRow = forwardRef<
     }, [panelIds, rowIndex, gridLayoutStateManager, renderPanelContents, setInteractionEvent]);
 
     return (
-      <>
+      <div ref={rowContainer}>
         {rowIndex !== 0 && (
-          <>
-            <EuiSpacer size="s" />
-            <EuiFlexGroup gutterSize="s">
-              <EuiButtonIcon
-                color="text"
-                aria-label={i18n.translate('kbnGridLayout.row.toggleCollapse', {
-                  defaultMessage: 'Toggle collapse',
-                })}
-                iconType={isCollapsed ? 'arrowRight' : 'arrowDown'}
-                onClick={toggleIsCollapsed}
-              />
-              <EuiTitle size="xs">
-                <h2>{rowTitle}</h2>
-              </EuiTitle>
-            </EuiFlexGroup>
-            <EuiSpacer size="s" />
-          </>
+          <GridRowSummary
+            isCollapsed={isCollapsed}
+            toggleIsCollapsed={toggleIsCollapsed}
+            rowTitle={rowTitle}
+          />
```

Comment:
Hmmm. Maybe `GridRowSummary` -> `InternalGridRow` or something like that? Or, we change the internal name of the current `GridRow` component to something like `GridRowWrapper` (still exporting it as `GridRow` 🤔

Reply:
I got inspired by the github markdown here: `<details><summary>Toggle me!</summary>Peek a boo!</details>`, but I am not strongly attached, will change :D 

<details><summary>Toggle me!</summary>Peek a boo!</details>

Reply:
Ahh! Makes sense 😆 I personally found it a bit confusing, because I wasn't sure what a "summary" was hahah. Maybe we could call it `GridRowHeader` instead? I don't mind the separation, the naming is just unclear IMO :)

Reply:
I agree, without the context summary for me is a 'below' item, not the 'above' one. `GridRowHeader` sounds good!